operators/gitlab-operator-kubernetes/0.17.1/manifests/gitlab-operator-kubernetes.clusterserviceversion.yaml

```diff
@@ -0,0 +1,100 @@
+      mediatype: image/png
+  install:
+    spec:
+      deployments: []
```

Comment:
@gl-distribution-oc this is very strange that it has empty value. Can you check it?

Reply:
@gl-distribution-oc  the PR is malformed. Multiple manifest items are missing. https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/2279 has the same issue.

Follow-up: https://gitlab.com/gitlab-org/cloud-native/gitlab-operator/-/issues/1086

Reply:
Addressed in 1d386ef6e8e1f41467ec50591ba42d0733b3a3dc